pubtools/iib/iib_ops.py

```diff
@@ -159,7 +120,7 @@
 }
 
 
-def push_items_from_build(build_details, state, pulp_repository):
+def push_items_from_build(build_details, state):
```

Comment:
I am Just curios: There is no other repository to which Pub would need to push data?

Reply:
The logic of where the index images are pushed is handled by pubtools-quay. Pubtools-iib doesn't do much by itself, just contacts IIB and returns its results. I guess its functionality was larger before pubtools-quay was written.